Leanplum-SDK/Classes/Leanplum.h

```diff
@@ -26,7 +26,6 @@
 #import <UserNotifications/UserNotifications.h>
 #import "LPInbox.h"
 #import "LPActionArg.h"
-#import "LPActionContext.h"
```

Comment:
This change actually breaks the SDK when you include it via Swift Package Manager. The `LPActionContext` class is no longer available.